webpack.dev.js

```diff
@@ -53,6 +57,15 @@ module.exports = mergeWithRules({
         filename: "[name].bundle.js",
     },
     plugins: [
-        new ReactRefreshWebpackPlugin(),
+        new ReactRefreshWebpackPlugin({
+            overlay: false,
```

Comment:
to disable the overlay provided by `@pmmmwh/react-refresh-webpack-plugin` to avoid conflicts.